### PR TITLE
chore(astro): Move all *.astro components under /components

### PR DESCRIPTION
**Pull request by panteliselef**
## Description
Move `@clerk/astro/components/*` to `@clerk/astro/components`

```diff
- import { UserProfile } from "@clerk/astro/components/interactive"
+ import { UserProfile } from "@clerk/astro/components"

- import { Protect } from "@clerk/astro/components/control"
+ import { Protect } from "@clerk/astro/components"

- import { SignInButton } from "@clerk/astro/components/unstyled"
+ import { SignInButton } from "@clerk/astro/components"
```

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by panteliselef** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @panteliselef - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/astro | 0.0.3-snapshot.v946ce16 |
| gatsby-plugin-clerk | 5.0.0-beta.45 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/astro`
```sh
npm i @clerk/astro@0.0.3-snapshot.v946ce16 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.0-beta.45 --save-exact
```

File: .changeset/fluffy-squids-fly.md
Comment (wobsoriano) [inline .changeset/fluffy-squids-fly.md:1]: Ahh this is so much easier to grasp

File: packages/astro/package.json [hunk 1]
```diff
@@ -67,13 +67,7 @@
...
     },
     "./env": "./env.d.ts",
-    "./components/control": "./components/control/index.ts",
-    "./components/control/*": "./components/control/*",
-    "./components/interactive": "./components/interactive/index.ts",
-    "./components/interactive/*": "./components/interactive/*",
-    "./components/unstyled": "./components/unstyled/index.ts",
-    "./components/unstyled/*": "./components/unstyled/*",
-    "./components/*": "./components/*",
+    "./components": "./components/index.ts",
```
Comment (wobsoriano) [inline packages/astro/package.json:70]: Ahh this is so much easier to grasp

File: packages/astro/tsconfig.json
Comment (wobsoriano) [inline packages/astro/tsconfig.json:1]: Just curious as to why we're excluding this folder?

**Review by wobsoriano — APPROVED** [review]
Just have a minor question but this is good to go

File: packages/astro/tsconfig.json
Comment (panteliselef) [reply packages/astro/tsconfig.json:1]: tsup does not seem to recognise the types from the component exported from a `*.astro` type.

Comment (panteliselef) [reply packages/astro/tsconfig.json:1]: Previously we has a `@ts-ignore` for each individual export, now i've moved this inside the tsconfig.

Comment (wobsoriano) [reply packages/astro/tsconfig.json:1]: thanks for answering!

**Pull request merged**